components/script/dom/mutationobserver.rs

```diff
@@ -37,4 +72,154 @@ impl MutationObserver {
         ScriptThread::add_mutation_observer(&*observer);
         Ok(observer)
     }
+
+    /// https://dom.spec.whatwg.org/#queue-a-mutation-observer-compound-microtask
+    /// Queue a Mutation Observer compound Microtask.
+    pub fn queueMutationObserverCompoundMicrotask(compoundMicrotask: Microtask) {
```

Comment:
Rather than accepting a Microtask argument here, this method should create it. The different callers that invoke this method do not need to customize any behaviour for the resulting microtask.